src/services/services.ts

```diff
@@ -2392,6 +2394,18 @@ module ts {
         }
 
         function synchronizeHostData(): void {
+            // perform fast check if host supports it
+            if (host.getProjectVersion) {
+                let hostProjectVersion = host.getProjectVersion();
+                if (hostProjectVersion) {

```

Comment:
I can but don't think that squeezing a few lines of code into one line buys us much. Also I'd like to be explicit about what is happening